### PR TITLE
[bugfix] Add installed_OS_is_certified OVAL for RHEL systems

### PR DESCRIPTION
**Pull request by redhatrises**
- Add installed_OS_is_certified OVAL for RHEL systems
- Update OVAL that requires FIPS to fail if installed_OS_is_certified is not valid

### CODE REVIEW

File: shared/oval/sshd_use_approved_ciphers.xml [hunk 1]
```diff
@@ -9,12 +9,15 @@
...
-      definition_ref="service_sshd_disabled" />
-      <criterion comment="Check ClientAliveInterval in /etc/ssh/sshd_config"
-      test_ref="test_sshd_use_approved_ciphers" />
+    <criteria operator="AND">
+      <extend_definition comment="Installed OS is certified" definition_ref="installed_OS_is_certified" />
+      <criteria comment="SSH is not being used or conditions are met"
+      operator="OR">
+        <extend_definition comment="sshd service is disabled"
+        definition_ref="service_sshd_disabled" />
+        <criterion comment="Check ClientAliveInterval in /etc/ssh/sshd_config"
```
Comment (mpreisler) [inline shared/oval/sshd_use_approved_ciphers.xml]: this comment is probably wrong

**Comment by redhatrises** [comment]
@mpreisler thanks. Looks like it was a leftover of a previous commit.

**Comment by mpreisler** [comment]
It was a nitpick but outdated comments make things confusing and OVAL is very confusing as it is :-D Thx for the fix! ACK

**Comment by shawndwells** [comment]
Love this. Thanks for creating this OVAL, Gabe!

**Comment by redhatrises** [comment]
@shawndwells np. XCCDF rule hopefully coming soon.

**Pull request merged**